ui/app/pages/swaps/view-quote/view-quote.js

```diff
@@ -472,7 +471,8 @@ export default function ViewQuote () {
             )}
             destinationDecimals={destinationTokenDecimals}
             destinationSymbol={destinationTokenSymbol}
-            isBestQuote={isBestQuote}
+            sourceIconUrl={selectedFromToken.iconUrl}
+            destinationIconUrl={destinationToken.iconUrl}
```

Comment:
Is iconUrl on tokens something added by swaps or did that exist prior?

Reply:
It's part of swaps tokens, but not other tokens in the app. You can see it on the objects returned by https://api.metaswap.codefi.network/tokens

Reply:
This will expose the user's IP address to the server hosting the token icon URL. This is why we bundle the icon with the extension for the list we have now